tests/Unit/Model/Export/ProductTest.php

```diff
@@ -155,6 +155,83 @@ public function testEmptyAttributeGroups($productType)
         $this->assertEmpty($exportModel->getAttributeGroups());
     }
 
+    /**
+     * @param array     $imageData
+     * @param string    $smallImagePath
+     *
+     * @covers ::setImages
+     * @dataProvider imageProvider
+     */
+    public function testImageSortOrder($imageData, $smallImagePath)
+    {
+        $productStub = $this->getProductDouble();
+        $productStub->method('getMediaGalleryImages')
+            ->will($this->returnValue($this->getTestCollection($imageData)));
```

Comment:
I cannot check as my phpunit is having issues, but supposedly can simplify to:
```suggestion
            ->willReturn($this->getTestCollection($imageData));
```